src/main/java/edu/wpi/first/pathweaver/PathWeaver.java

```diff
@@ -35,10 +41,27 @@ public void start(Stage primaryStage) throws IOException {
     primaryStage.setScene(this.mainScene);
     primaryStage.show();
     Loggers.setupLoggers();
+    var warn = new Alert(AlertType.WARNING);
+    warn.setTitle("PathWeaver is deprecated!");
+    warn.setHeaderText("PathWeaver is deprecated and will be removed by 2027.");
+    warn.setContentText("Consider using Choreo or PathPlanner instead.");
+    ButtonType learnMore = new ButtonType("Learn more");
+    ButtonType dontCare = new ButtonType("Don't care");
```

Comment:
Let's try to get some better wording here. Don't care is kind of not great.

Reply:
Switched to "Ok".